### PR TITLE
Use FieldKey

### PR DESCRIPTION
**Pull request by Maxim-Inv**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.1
+// swift-tools-version:5.2
```
Comment (m-barthelemy) [inline Package.swift:1]: I was too lazy to change the format to 5.2
Thank you so much for doing it :)

File: Sources/QueuesFluentDriver/JobModel.swift [hunk 2]
```diff
@@ -10,6 +10,16 @@ public enum QueuesFluentJobState: String, Codable, CaseIterable {
...
+extension FieldKey {
+    static var key: Self { "key" }
+    static var data: Self { "data" }
+    static var state: Self { "state" }
+
+    static var createdAt: Self { "created_at" }
+    static var updatedAt: Self { "updated_at" }
+    static var deletedAt: Self { "deleted_at" }
+}
+
```
Comment (m-barthelemy) [inline Sources/QueuesFluentDriver/JobModel.swift:22]: It's cleaner now with this `FieldKey` extension, thanks.

File: Package.swift [hunk 3]
```diff
@@ -17,16 +17,16 @@ let package = Package(
         .package(url: "https://github.com/vapor/vapor.git", from: "4.0.1"),
         .package(url: "https://github.com/vapor/fluent.git", from: "4.0.0-rc.2"),
         .package(url: "https://github.com/vapor/sql-kit.git", from: "3.0.0-rc"),
-        .package(url: "https://github.com/vapor/queues.git", from: "1.0.0"),
+        .package(url: "https://github.com/vapor/queues.git", from: "1.0.0-rc"),
```
Comment (m-barthelemy) [inline Package.swift:20]: Was hoping Queues would go out of RC a few days ago, hasn't happened yet. Let's switch back to `-rc` for now.

Comment (Maxim-Inv) [reply Package.swift:20]: Ok

**Pull request merged**